lib/main.sh

```diff
@@ -404,6 +404,7 @@ if [[ $IGNORE_UPDATES != yes ]]; then
 	fetch_from_repo "https://github.com/MarvellEmbeddedProcessors/binaries-marvell" "marvell-binaries" "branch:binaries-marvell-armada-18.12"
 	fetch_from_repo "https://github.com/armbian/odroidc2-blobs" "odroidc2-blobs" "branch:master"
 	fetch_from_repo "https://github.com/armbian/testings" "testing-reports" "branch:master"
+	fetch_from_repo "https://github.com/rockchip-linux/rkbin" "rkbin" "branch:master"
```

Comment:
I would not rely on Rockchip's rkbin repository.
Binaries come and go there and one they the image build will stop working because `rk3399_ddr_800MHz_v1.24.bin` will be replaced by `rk3399_ddr_800MHz_v1.25.bin` ;-)

You could bring missing binaries and config files into https://github.com/armbian/rkbin, which needs some cleaning now but will definitely be more stable.

Reply:
armbian/rkbin and rockchip/rkbin are different. I don't want to study how to build EDGE bootload from armbian/rkbin

Reply:
fix this by using *